pkg/cli/cliflags/flags.go

```diff
@@ -255,8 +255,11 @@ production usage.`,
 	ServerInsecure = FlagInfo{
 		Name: "insecure",
 		Description: `
-Start an insecure node, using unencrypted (non-TLS) connections. This is strongly discouraged for
-production usage.`,
+Start an insecure node, using unencrypted (non-TLS) connections,
+listening on all IP addresses and disabling password authentication
```

Comment:
"listening on all IP addresses" isn't true - that's controlled by the host flag, as recently changed by Marc's certs overhaul.